SU-SWS/stanford_profile_helper
# D8CORE-1609: Super Footer and Global Messages

# READY FOR REVIEW 

# Summary
- Styles for Super Footer
- Styles for Global Messages
- Updates to packages.

# Needed By (Date)
- Sept 3rd

# Urgency
- High

# Steps to Test

1. Review code
2. Do a build of cardinalsites or lelandd8 with the D8CORE-1609 branch on those stacks
3. Install drupal (blt di)
4. Clear all caches
5. Create content for the global messages and super footer
6. See that they are styled
7. Review mobile styles

# Affected Projects or Products
- D8CORE
- Adapt
- SOE

# Associated Issues and/or People
- D8CORE-2490
- D8CORE-1609

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
